galpy/df/streamspraydf.py

```diff
@@ -102,11 +105,18 @@ def __init__(
             self._center.integrate(self._progenitor_times, self._centerpot)
         else:
             self._center = None
+        if progpot is not None:
+            progtrajpot = MovingObjectPotential(
+                orbit=self._progenitor,
+                pot=progpot,
+                ro=self._ro,
+                vo=self._vo,
+            )
+            self._pot = flatten_potential([self._pot, progtrajpot])
```

Comment:
Given that ``self._pot`` is already flattened, this can just be
```suggestion
            self._pot = self._pot+progtrajpot
```